elb/elb.go

```diff
@@ -120,18 +124,26 @@ func (e *elb) Start() error {
 	return nil
 }
 
-func (e *elb) findFrontEndElbs() ([]string, error) {
+// FindFrontEndElbs find all the frontend elbs.
```

Comment:
nit "finds"

Can we add how it does it rather than just repeating the method name as a sentence :)